patch/0093-hwmon-pmbus-Add-support-for-MPS-Multi-phase-mp2855-c.patch

```diff
@@ -0,0 +1,457 @@
+From ad3bc878f59106091171e2870b7613b74e021691 Mon Sep 17 00:00:00 2001
+From: Vadim Pasternak <vadimp@nvidia.com>
+Date: Thu, 4 Jan 2024 09:12:29 +0000
+Subject: [PATCH v6.1 9/9] hwmon: (pmbus) Add support for MPS Multi-phase
+ mp2855 controller
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Add support for MP2855 device, which is a dual-loop, digital,
+multi-phase controller that provides power for the core of the
+AMD SVI2 2.0 platform.
+
+Device configurations and fault parameters can be configured or
+monitored via the PMBus interface.
+
+The device:
+- Can work with MPS’s Intelli-PhaseTM products
+- Can be configured with up to 9-phase operation for rail 1 and up
+  to 4-phase operation for rail 2.
+- Provides an on-chip non-volatile memory (NVM) to store and restore
+  device configurations.
+- Can monitor and report the output current (IOUT) through the CS
+  output from Intelli-PhaseTM products.
+
+The device is available in a TQFN-40 (5mmx5mm) package.
+
+Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>
+---
+ drivers/hwmon/pmbus/Kconfig  |  10 +
+ drivers/hwmon/pmbus/Makefile |   1 +
+ drivers/hwmon/pmbus/mp2855.c | 380 +++++++++++++++++++++++++++++++++++
+ 3 files changed, 391 insertions(+)
+ create mode 100644 drivers/hwmon/pmbus/mp2855.c
+
+diff --git a/drivers/hwmon/pmbus/Kconfig b/drivers/hwmon/pmbus/Kconfig
+index 77d67344cee4..f173c4195d86 100644
+--- a/drivers/hwmon/pmbus/Kconfig
++++ b/drivers/hwmon/pmbus/Kconfig
+@@ -290,6 +290,16 @@ config SENSORS_MAX8688
+ 	  This driver can also be built as a module. If so, the module will
+ 	  be called max8688.
+ 
++config SENSORS_MP2855
++	tristate "MPS MP2855"
++	help
++	  If you say yes here you get hardware monitoring support for MPS
++	  MP2855 Daula-Loop, Digital, Multi-Phase Controller with PMBus
```

Comment:
Nitpicking, but possible typo in "Daula"

Reply:
Handled